### ISSUE TITLE
Arbitrary Code Execution audit error

### ISSUE THREAD

**Issue by kpotter-m2** [forwardemail/email-templates#425]
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Arbitrary Code Execution                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ underscore                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=1.12.1                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ email-templates                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ email-templates > @ladjs/i18n > country-language >           │
│               │ underscore                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1674                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Arbitrary Code Execution                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ underscore                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=1.12.1                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ email-templates                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ email-templates > @ladjs/i18n > i18n-locales >               │
│               │ country-language > underscore                                │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1674                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by kenbkop** [comment]
Yes, I am running into this also.  Hopefully this can be fixed soon.

**Comment by niftylettuce** [comment]
This is not really a big issue but I'll bump it at some point soon

**Comment by niftylettuce** [comment]
v8.0.5 released https://github.com/forwardemail/email-templates/releases/tag/v8.0.5

**Comment by niftylettuce** [comment]
Sorry not yet fixed - I had to do one more, will be in v8.0.6

**Comment by niftylettuce** [comment]
v8.0.6 released https://github.com/forwardemail/email-templates/releases/tag/v8.0.6
